tests/suites/test_suite_psa_crypto.function

```diff
@@ -16,6 +16,9 @@
 #include "test/psa_crypto_helpers.h"
 #include "test/psa_exercise_key.h"
 
+/* If this comes up, it's a bug in the test code or in the test data. */
+#define UNUSED 0xdeadbeef
```

Comment:
Where else should it be used? I think we mostly put `0` when some numerical test data isn't used, but I thought it would be useful in the `derive_input` tests to make it more apparent when some parameters are not used, due to the unusually large number of parameters.

Locating where else to start using `UNUSED` is a long task and out of scope, so I'd rather not do it.

There may be places where the same numerical value is used for other purposes (it's one of the classics of words-in-hex) but it wasn't used for unused test data before.

Reply:
I meant that there are lots of occurrences of 0xdeadbeef in this file, but yes, it's actually in 11 files here and addressing all of them would be well out of scope. None of the rest use a #define so I think the #define could be removed if we want to be as consistent as possible, but also fine as-is.